Rmd/stats_report.Rmd

```diff
@@ -1,31 +1,30 @@
 ---
 params:
   projectdir: null # Use when loading. If null, here::here() will be used. Can also hard code if for some reason you need that
-  project_title: "Default"    # Change depending on your project name
+  project_title: "Liver toxic chemicals"    # Change depending on your project name
   project_name: "Your name here" 
   project_description: null # description of project for report, optional
-  batch_var: "Batch"          # For example, multiple plates
-  dose: "Dose"               # If there is a dose in the experiment; otherwise use NULL
+  batch_var: "batch"          # For example, multiple plates
+  dose: "dose"               # If there is a dose in the experiment; otherwise use NULL
   platform: "TempO-Seq"      # TempO-Seq Or RNA-Seq
   nmr_threshold: 100000    # 10% of 1M reads for TempOSeq = 100,000; 10% of 10M reads for RNA-Seq = 1,000,000.
-  write_additional_output: FALSE # export BMD, biomarker, and biosets output? facet variable should contain chemical and timepoint information
-  celltype: "TK6 cells" # only required for biosets output. Cell type or species used
+  write_additional_output: TRUE # export BMD, biomarker, and biosets output? facet variable should contain chemical and timepoint information
+  celltype: "Liver spheroids" # only required for biosets output. Cell type or species used
   units: "uM" # only required for biosets output. Units of dose
   species: "human"                 # one of human, mouse, rat, hamster
-  design: "Group"                # single experimental group of interest; entries in this column must match the contrast names.
-  intgroup: ["Group"]            # experimental group of interest plus covariates; can be more than one
-  group_facet: "Chemical_S9"       # If you have many different experimental groups, you may subset the report by specifying a column in the metadata to filter groups, and then setting the group of interest in group_filter
+  design: "group"                # single experimental group of interest; entries in this column must match the contrast names.
+  intgroup: ["group"]            # experimental group of interest plus covariates; can be more than one
+  group_facet: "chemical_timepoint"       # If you have many different experimental groups, you may subset the report by specifying a column in the metadata to filter groups, and then setting the group of interest in group_filter
   group_filter:  null        # Which group will this report be done on?
-  display_group_facet: "Chemical_S9" #  
-  display_group_filter:  "AMP +S9" # 
-  sortcol: "Dose"                # Optionally, a column by which to sort the contrasts
+  display_group_facet: "chemical_timepoint" #  
+  display_group_filter:  "estradiol 10d" # 
+  sortcol: "dose"                # Optionally, a column by which to sort the contrasts
```

Comment:
Some day we will have to figure out a way to remember not to commit these changes...
I bet there's a way to automate warnings

Reply:
But today is not that day!